.github/BE_COVENTION_CHECKLIST.md

```
@@ -0,0 +1,126 @@
+# 백엔드 컨벤션 리뷰 체크리스트
+
+이 문서는 [BE 코드 컨벤션 Wiki](https://github.com/woowacourse-teams/2025-moitz/wiki/%5BBE%5D-%EC%BD%94%EB%93%9C-%EC%BB%A8%EB%B2%A4%EC%85%98)를 기반으로 작성된 코드 리뷰 체크리스트입니다.
+
+## 📋 목차
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)